lua/gruvbox-baby/colors.lua

```diff
@@ -92,6 +92,12 @@ function M.config(config)
     }
     colors = vim.tbl_extend("force", colors, transparent)
   end
+
+  if config.overrides then
+    for override_color, new_color in pairs(config.overrides) do
+      colors[override_color] = new_color
+    end
+  end
   return colors
```

Comment:
you missed changing the `overrides` here 😉

Reply:
no wonder it didn't work lol